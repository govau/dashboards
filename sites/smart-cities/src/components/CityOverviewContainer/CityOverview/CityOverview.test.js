import React from 'react';
import { shallow } from 'enzyme';
import CityOverview from './CityOverview';

jest.mock('../../CategoryOverviews/CategoryOverviews', () => 'CategoryOverviews');
jest.mock('../../PageBanner/PageBanner', () => 'PageBanner');

const defaultProps = {
  city: {
    name: 'Perth',
    description: 'Perth',
    id: 'perth',
  },
};

it('should match Snapshot', () => {
  const component = shallow(
    <CityOverview {...defaultProps} />
  );

  expect(component.debug()).toMatchSnapshot();
});
