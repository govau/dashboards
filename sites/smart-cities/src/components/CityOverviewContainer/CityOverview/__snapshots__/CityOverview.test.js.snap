// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Snapshot 1`] = `
"<PageWrapper cityId=\\"perth\\" cityName=\\"Perth\\" categoryColorName=\\"primary\\" categoryId=\\"overview\\">
  <PageBanner colorName=\\"primary\\" description=\\"Perth\\" indicatorId=\\"population\\" indicatorSubCategory={{...}} title=\\"Perth\\" city={{...}} />
  <CategoryOverviews city={{...}} />
</PageWrapper>"
`;
